cocos/2d/CCFontAtlasCache.cpp

```diff
@@ -262,4 +264,54 @@ void FontAtlasCache::unloadFontAtlasTTF(const std::string& fontFileName)
     }
 }
 
+std::string FontAtlasCache::getCachedTextureInfo() {
+    std::stringstream ss;
+    char buf[4096];
```

Comment:
Please remove buf add white directly to ss

Reply:
Variable `buf` used as temporary buffer within safe `snprintf`, while stringstream as manager allocator of plain memory (to avoid reallocation memory while appender of `std::string`)